sunpy-dev-env.yml

```diff
@@ -2,6 +2,13 @@ channels:
   - conda-forge
 
 dependencies:
+  # Build time (taken from pyproject.toml)
+  - setuptools
+  - setuptools-scm
+  - extension-helpers
+  - pip
+  - numpy
+
```

Comment:
Er, all of these dependencies are already explicitly specified later in the environment file, just not grouped in one place: `numpy` and `setuptools` under "Required" and the rest under "Installation".  Do you really want to have these dependencies specified twice?

Reply:
It didn't work anyway, so I could just pull this change I think.